### PR TITLE
 Introduce --toolkit-dir to override toolkit directory during coding

### PR DESCRIPTION
**Pull request by vchudnov-g**
This allows running the artman command while incorporating changes to a local clone of the googleapis/toolkit repository.

### CODE REVIEW

File: artman/cli/main.py [hunk 1]
```diff
@@ -152,6 +152,15 @@ def parse_args(*args):
...
     )
+    parser.add_argument(
+        '--toolkit-dir',
+        type=str,
+        default='',
+        help='[Optional] Local directory containing the local clone of the '
+        'github/toolkit repo to use, instead of the one specified in the '
+        'configuration files. This flag is useful for testing development '
+        'changes to toolkit via an artman invocation.',
+    )
```
Comment (ethanbao) [inline artman/cli/main.py]: Were you suggesting to mount the local toolkit to the docker container and use that inside the docker container?

Comment (vchudnov-g) [reply artman/cli/main.py]: Yes, exactly. I can mount via "-v .../toolkit/:/tmp/toolkit" to Docker and then run artman inside it with "--toolkit-dir /tmp/toolkit"

**Comment by vchudnov-g** [comment]
BTW, @ethanbao , the CI error message `ImportError: No module named 'msgpack'` appears to be unrelated to my changes. Any ideas what could be causing it?

**Comment by ethanbao** [comment]
msgpack error should have been fixed by https://github.com/googleapis/artman/commit/d637f72d1b928b803bb652978364544b510a04c0#diff-b4ef698db8ca845e5845c4618278f29a

Fetch the latest repo and that error should be gone.

File: artman/cli/main.py (hunk 1 above)
Comment (ethanbao) [reply artman/cli/main.py]: In ~/.artman/config.yaml, there is local_paths.toolkit which will point artman to local toolkit dir. But that only works for local mode. If you are okay with mount via -v, you can simply update the ~/.artman/config.yaml inside the docker image without introducing this new flag.

Unlike --local-repo-dir that was recently added, I feel this --toolkit-dir will be even less used by normal users.

Comment (vchudnov-g) [reply artman/cli/main.py]: Ah, OK. I couldn't this to work by changing the toolkit path in my non-Docker .artman.config.yaml. But changing it inside the Docker container works.
(We may want to think about propagating that information in automatically.)
I agree, in that case this PR is not necessary. I'll close it.

**Pull request closed without merging**